crates/symbolicator/src/services/symbolication.rs

```diff
@@ -130,76 +130,99 @@ impl<'a> SymCacheLookupResult<'a> {
 /// The CFI modules referenced by a minidump for CFI processing.
 ///
 /// This is populated from the [`CfiCacheResult`], which is the result of looking up the
-/// modules determined to be referenced by this minidump by
-/// [`SymbolicationActor::get_referenced_modules_from_minidump`].  It contains the CFI cache
+/// modules determined to be referenced by this minidump.  It contains the CFI cache
```

Comment:
Isn't this statement `which is the result of looking up the modules determined to be referenced by this minidump` no longer true with your refactoring?  I think you create one cache which you keep adding to during the various iterations.  But you can end up with unused items in this cache now since a later iteration might no longer think it requires them.